level-2/행렬테두리회전하기/0imbean0.js

```diff
@@ -0,0 +1,61 @@
+function getSquare(rows, columns) {
+  return Array.from({ length: rows }).map((_, row) => {
+    return Array.from({ length: columns }).map((_, column) => {
+      return columns * row + (column + 1);
+    });
+  });
+}
+
+function rotation(square, query) {
+  const [x1, y1, x2, y2] = query;
+  const dx = x2 - x1;
+  const dy = y2 - y1;
+  const command = [
+    [0, 1, dy],
+    [1, 0, dx],
+    [0, -1, dy],
+    [-1, 0, dx],
+  ];
+
+  let mX = x1 - 1;
+  let mY = y1 - 1;
+  let prevNum = square[mX][mY];
+
+  const lotationNums = [];
+
+  command.forEach(([x, y, range]) => {
+    Array.from({ length: range }).forEach(() => {
+      mX += x;
+      mY += y;
+
+      const nextNum = square[mX][mY];
+
+      square[mX][mY] = prevNum;
+      prevNum = nextNum;
+
+      lotationNums.push(nextNum);
+    });
+  });
+
+  return lotationNums;
+}
+
+function getMinimun(nums) {
+  let min = 9999999999;
+
+  nums.forEach((num) => {
+    min = min < num ? min : num;
+  });
+
+  return min;
```

Comment:
```suggestion
 return Math.min(...nums)
```

내장 객체 메서드를 활용하면 간단하게 작성 가능할거 같아용!

- [Math.min() / mdn](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/Math/min)

Reply:
> const getMinimum = (arr) => arr.reduce((a, b) => Math.min(a, b));